srcpkgs/afew/template

```diff
@@ -0,0 +1,28 @@
+# Template file for 'afew'
+pkgname=afew
+version=1.3.0
+revision=1
+noarch=yes
+build_style=python3-module
+pycompile_module="afew"
+hostmakedepends="git python3-setuptools python3-Sphinx pkg-config"
+depends="python3-setuptools notmuch-python3 python3-dkimpy"
+short_desc="An initial tagging script for notmuch mail"
+maintainer="Alexander Gehrke <void@qwertyuiop.de>"
+license="ISC"
+homepage="https://github.com/afewmail/afew"
+_gitsrc="${homepage}.git"
+
+do_fetch() {
+	git clone ${_gitsrc} ${wrksrc} --branch ${version}
```

Comment:
No. Use the release tarball instead of git.

Reply:
Using the release tarball causes a build failure with a message saying that github's release tarballs do not contain some required metadata and a clone should be used.

Reply:
ok, I checked Gentoo's ebuild, the Pipy mirror seems to contain all required data for the build, updated it.